yb-voyager/cmd/analyzeSchema.go

```diff
@@ -297,7 +300,21 @@ func addSummaryDetailsForIndexes() {
 	exportedIndexes := summaryMap["INDEX"].objSet
 	unexportedIdxsMsg := "Indexes which are neither exported by yb-voyager as they are unsupported in YB and needs to be handled manually:\n"
 	unexportedIdxsPresent := false
+
+	// List for assessment report has suffix " INDEX" added to the original index type
+	suffix := " INDEX"
+	oracleUnsupportedIndexTypesForAnalyze := lo.Map(OracleUnsupportedIndexTypes, func(s string, _ int) string {
+		if strings.HasSuffix(s, suffix) {
+			return strings.TrimSuffix(s, suffix)
+		}
+		return s
```

Comment:
Should we even return the ones that don't have ` INDEX` suffix in the `OracleUnsupportedIndexTypes`?